hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/io/compress/TestCodec.java

```diff
@@ -882,26 +908,28 @@ private void testGzipCodecWrite(boolean useNative) throws IOException {
 
     BufferedWriter w = null;
     Compressor gzipCompressor = CodecPool.getCompressor(codec);
-    if (null != gzipCompressor) {
-      // If it gives us back a Compressor, we should be able to use this
-      // to write files we can then read back with Java's gzip tools.
-      OutputStream os = new CompressorStream(new FileOutputStream(fileName),
-          gzipCompressor);
-      w = new BufferedWriter(new OutputStreamWriter(os));
-      w.write(msg);
-      w.close();
-      CodecPool.returnCompressor(gzipCompressor);
-
-      verifyGzipFile(fileName, msg);
-    }
-
-    // Create a gzip text file via codec.getOutputStream().
```

Comment:
maybe we should keep this test still.

Reply:
Hmm? you mean keeping original single write?

Reply:
oh I mean the original test with comment "// Create a gzip text file via codec.getOutputStream()." I think we should change it to use multi-write too.

Reply:
Oh, got it. Yea, removed it accidentally. Restoring it with multi-write.